core/src/main/java/org/frankframework/pipes/MessageSendingPipe.java

```diff
@@ -556,13 +489,8 @@ private PipeRunResult sendMessageWithRetries(Message input, Message originalMess
 
 		correlationID = logToMessageLog(input, session, originalMessage, messageID, correlationID);
 
-		if (getListener() != null) {
-			Message result = listenerProcessor.getMessage(getListener(), correlationID, session);
-			sendResult.setResult(result);
-		}
-		if (Message.isNull(sendResult.getResult())) {
-			sendResult.setResult(new Message(""));
-		}
```

Comment:
If null -> empty string change has been removed, this was here because of the Larva.